bundles/org.openhab.binding.icloud/src/main/java/org/openhab/binding/icloud/internal/json/response/ICloudAccountDataResponse.java

```diff
@@ -15,13 +15,15 @@
 import java.util.List;
 
 import com.google.gson.annotations.SerializedName;
+import org.eclipse.jdt.annotation.NonNullByDefault;
 
 /**
  * Serializable class to parse the device information json response
  * received from the Apple server.
  *
  * @author Patrik Gfeller - Initial Contribution
  */
+@NonNullByDefault
```

Comment:
I think adding `@NonNullByDefault` to DTO classes is not necessary and avoids your problems.

Reply:
Ok I will remove these. I just added from the Travis warnings, didn't checked that it might be needed or not :)